### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by dfarrow0**
- make covidcast more prominent
- trim down other content
- move references to new page

### CODE REVIEW

**Comment by dfarrow0** [comment]
**note** this got stacked on top of my other documentation PR, #252 -- suggest merging that one first.

---

@capnrefsmmat @ryantibs please take a look

File: README.md [hunk 1]
```diff
@@ -8,26 +8,19 @@ This is the home of [Delphi](https://delphi.cmu.edu/)'s epidemiological data
...
+which offers a number of data streams relating to the ongoing COVID-19
+pandemic and powers the [COVIDcast project](https://covidcast.cmu.edu/).
 
-For a list of many other data sources relevant to COVID-19, publicly available
-through external sites, we have compiled a simple
-[spreadsheet](https://docs.google.com/spreadsheets/d/16Nn_3ZvSLnpxRyA2DkoMMzyrd11-AlGJXasS0owln88/edit#gid=0).
+Our other, non-COVID data sources remain generally available on a best-effort
+basis. However, non-COVID sources should be held to scrutiny at this time since
+they were designed to serve as indicators of typical seasonal ILI
+(influenza-like illness), not pandemic ILI or CLI (covid-like illness).
```
Comment (ryantibs) [inline README.md:21]: Small nitpick: "COVID-like illness"

File: README.md [hunk 2]
```diff
@@ -52,21 +45,5 @@ like to cite it, we would gratefully recommend the following copy:
...
-   Surveillance using Twitter." AAAI Conference on Artificial
-   Intelligence. 2014.
- - Generous, Nicholas, et al. "Global disease monitoring and forecasting with
-   Wikipedia." (2014): e1003892.
- - Hickmann, Kyle S., et al. "Forecasting the 2013–2014 Influenza Season Using
-   Wikipedia." (2015): e1004239.
- - McIver, David J., and John S. Brownstein. "Wikipedia usage estimates
-   prevalence of influenza-like illness in the United States in near real-time."
-   PLoS Comput Biol 10.4 (2014): e1003581.
+See [related work](docs/related_work.md) for a sample of additional work in
```
Comment (ryantibs) [inline README.md:47]: Could consider linking to the COVIDcast API blog post: https://delphi.cmu.edu/blog/2020/10/07/accessing-open-covid-19-data-via-the-covidcast-epidata-api/ as general friendly intro.  Either here or in the COVIDcast API docs?

**Comment by ryantibs** [comment]
Looks good, thank you @dfarrow0.  I left two very small comments.  

Since this builds off your other doc update #252, I suggest @krivard (who is the reviewer for that one) merges this one, after she merges that one.

File: README.md (hunk 1 above)
Comment (dfarrow0) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (dfarrow0) [reply README.md:50]: added a link to the blog post higher up in the page, in the COVIDcast section

**Comment by dfarrow0** [comment]
Thanks Ryan, made the edits you suggested.

@krivard merge when ready

**Pull request merged**